cirq-google/cirq_google/engine/simulated_local_job.py

```diff
@@ -55,6 +57,10 @@ def __init__(
         self._type = simulation_type
         self._failure_code = ''
         self._failure_message = ''
+        if self._type == LocalSimulationType.ASYNCHRONOUS:
+            # If asynchronous mode, just kick off a new task and move on.
+            self._thread = concurrent.futures.ThreadPoolExecutor(max_workers=1)
```

Comment:
should we be concerned about closing this threadpool? I usually see it used as a context manager which means there should be some sort of try..finally

Reply:
Added a call to shutdown immediately, since this will clean up the resources once the future is complete.